frontend/src/components/Navbar.js

```diff
@@ -15,29 +16,58 @@ const Navbar = () => {
             <button className="hamburger-icon" onClick={() => setVisible((visible) => !visible)}>
                 <img src={icon} alt="hamburger-icon" />
             </button>
+            {
+                localStorage.getItem('TOKEN') ? (
+                    <ul className={visible === true ? ("nav-list") : (" nav-list nav-list-none")}>
 
-            <ul className={visible === true ? ("nav-list") : (" nav-list nav-list-none")}>
-
-                <Link to={{ pathname: '/products' }}>
-                    <li className="badge-relative" onClick={() => setVisible((visible) => !visible)}>
-                        Products
-                    </li>
-                </Link>
-
-                <Link to={{ pathname: '/wishlist' }}>
-                    <li className="badge-relative" onClick={() => setVisible((visible) => !visible)}>
-                        WishList
-                        <span className="badge-nav-wishlist">{state.wishlist.length}</span>
-                    </li>
-                </Link>
-                <Link to={{ pathname: '/cart' }}>
-                    <li className="badge-relative" onClick={() => setVisible((visible) => !visible)}>
-                        Cart
-                        <span className="badge-nav-cart">{state.cart.length}</span>
-                    </li>
-                </Link>
-            </ul>
+                        <Link to={{ pathname: '/products' }}>
+                            <li className="badge-relative" onClick={() => setVisible((visible) => !visible)}>
+                                Products
+                        </li>
+                        </Link>
+
+                        <Link to={{ pathname: '/wishlist' }}>
+                            <li className="badge-relative" onClick={() => setVisible((visible) => !visible)}>
+                                WishList
+                            <span className="badge-nav-wishlist">{state.wishlist.length}</span>
+                            </li>
+                        </Link>
+                        <Link to={{ pathname: '/cart' }}>
+                            <li className="badge-relative" onClick={() => setVisible((visible) => !visible)}>
+                                Cart
+                            <span className="badge-nav-cart">{state.cart.length}</span>
+                            </li>
+                        </Link>
+                        <Link to={{ pathname: '/' }}>
+                            <li className="badge-relative" onClick={() => {
+                                setVisible((visible) => !visible)
+                                dispatch({ type: USER_LOGOUT })
+                            }}>
+                                <i className="fas fa-sign-out-alt"></i>
```

Comment:
This icon is not aligned properly. The cart value is being displayed on it.
![image](https://user-images.githubusercontent.com/30731236/117129240-0f795980-adbc-11eb-867e-f6cef1a82a0c.png)

Reply:
done